prow/monitoring/grafana_expose.yaml

```diff
@@ -0,0 +1,29 @@
+---
+apiVersion: v1
+kind: Service
+metadata:
+  labels:
+    app: grafana
+  name: grafana
+  namespace: prow-monitoring
+spec:
+  ports:
+  - name: http
+    port: 3001
+    targetPort: http
+  selector:
+    app: grafana
+---
+apiVersion: networking.k8s.io/v1beta1
+kind: Ingress
+metadata:
+  name: grafana
+  namespace: prow-monitoring
+spec:
+  rules:
+    - host: grafana-prow-monitoring.k8s.io
```

Comment:
I think we would need to copy the `tls-ing_ingress.yaml` but using `monitoring.prow` instead of `prow` (e.g. `monitoring.prow.k8s.io`

Reply:
Let's also cross-link the PR to change the DNS record

Reply:
> Let's also cross-link the PR to change the DNS record
Is this the right next step? How/when do we get the IP for the A record?
https://github.com/kubernetes/k8s.io/blob/a3a005fa33a7403624629f0994621203fa78b07a/dns/zone-configs/k8s.io.yaml#L191-L193

Reply:
I think i found [the answer](https://cloud.google.com/kubernetes-engine/docs/tutorials/configuring-domain-name-static-ip).

Reply:
I guess the secret `grafana-tls` will [be created automatically](https://blog.jetstack.io/blog/kube-lego/) by annotation `kubernetes.io/tls-acme: "true"`:

https://github.com/kubernetes/test-infra/blob/0b01f1ecf038b855eb2b6d318063a76be5fffab0/prow/monitoring/grafana_expose.yaml#L30